charmcraft.yaml

```diff
@@ -23,25 +23,10 @@ parts:
       - files
       - mod
       - requirements.txt
-
-bases:
-  - build-on:
-    - name: ubuntu
-      channel: "22.04"
-      architectures: ["amd64", "arm64"]
-    run-on:
-      - name: ubuntu
-        channel: "24.04"
-        architectures:
-          - amd64
-          - arm64
-      - name: ubuntu
-        channel: "22.04"
-        architectures:
-          - amd64
-          - arm64
-      - name: ubuntu
-        channel: "20.04"
-        architectures:
-          - amd64
-          - arm64
+platforms:
+  ubuntu@20.04:amd64:
+  ubuntu@22.04:amd64:
+  ubuntu@24.04:amd64:
```

Comment:
We should add a noble bundle to the functional tests, so the noble charm is also tested.